tests/addChallenge/endpoint.test.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+var test = require('tape');
+var initServer = require('../../example/server.js');
+var initialTablesData = require('../../fixtures/importMockData.js');
+
+var config = require('../../config/load-config.js');
+
+
```

Comment:
I'm not sure if this test file is doing anything at the moment as the test is commented